packages/router/index.js

```diff
@@ -173,6 +211,13 @@ const pathProxy = (req, res, next) => {
 
 
 function isInternalRequest(req, res, next) {
+
+log.info({
```

Comment:
@arkaprovob I appreciate all the INFO logs that you added for debugging purposes, don't you think we should be adding in some checks as well to indicate any ERRORs. wdyt?